install/020-MqttBridgeConfigMap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mqtt-bridge-config
+  labels:
+        app: mqtt-bridge
+data:
+  application.properties: |
+    #Bridge related settings
+    bridge.id=my-bridge
+
+    #MQTT server common
+    mqtt.server.host=localhost
+    mqtt.server.port=1883
+    #Apache Kafka common
+    kafka.bootstrap.servers=localhost:9092
```

Comment:
maybe we should have a placeholder here as well?  We don't know where the Kafka cluster is placed.

Reply:
just use a placeholder for the address and leave 9092 as following port.

Reply:
well no, actually it can be a comma separated list of Kafka brokers so even this one doesn't work fine.
Let's just use `<kafka-bootstrap-servers>` ... who uses Kafka knows what to put there.

Reply:
Completely agree.